documentation/asciidoc/accessories/camera/libcamera_options_common.adoc

```diff
@@ -19,6 +19,35 @@ libcamera-apps build: ca559f46a97a 27-09-2021 (14:10:24)
 libcamera build: v0.0.0+3058-c29143f7
 ----
 
+----
+	--list-cameras			List the cameras available for use
+----
+
+The `--list-cameras` will display the available cameras attached to the board that can be used by the application. This option also lists the sensor modes supported by each camera. For example:
+
+----
+Available cameras
+-----------------
+0 : imx219 [3280x2464] (/base/soc/i2c0mux/i2c@1/pca@70/i2c@0/imx219@10)
+    Modes: 'SRGGB10_CSI2P' : 640x480 1640x1232 1920x1080 3280x2464 
+           'SRGGB8' : 640x480 1640x1232 1920x1080 3280x2464 
+1 : imx477 [4056x3040] (/base/soc/i2c0mux/i2c@1/pca@70/i2c@1/imx477@1a)
+    Modes: 'SRGGB10_CSI2P' : 1332x990 
+           'SRGGB12_CSI2P' : 2028x1080 2028x1520 4056x3040 
+----
+
+In the above example, the IMX219 sensor is available at index 0 and IMX477 at index 1. The sensor mode identifier takes the following form:
+----
+<Bayer order><Bit-depth>_<Optional packing> : <Resolution list>
+----
+For the IMX219 in the above example, all modes have a `RGGB` Bayer ordering and provide either 8-bit or 10-bit CSI2 packed readout at the listed resolutions.
+
+----
+	--camera			Selects which camera to use <index>
+----
+
+The `--camera` option will select which camera to use from the supplied <index> value. The <index> value can be obtained from the `--list-cameras` option.
```

Comment:
Presumably this defaults to 0 if not specified?

Reply:
Correct!